domain/approval.go

```diff
@@ -26,6 +26,10 @@ type Approval struct {
 	UpdatedAt time.Time `json:"updated_at"`
 }
 
+func (a *Approval) IsHumanApproval() bool {
```

Comment:
Pardon my lack of context, but would `IsManualApproval` better here?

Reply:
that works, changing it 👍